src/parser.rs

```diff
@@ -168,6 +168,10 @@ impl Parse for BrushMode {
                     "xsym" => Ok((BrushMode::XSym, p)),
                     "ysym" => Ok((BrushMode::YSym, p)),
                     "xray" => Ok((BrushMode::XRay, p)),
+                    "line" => optional(whitespace())
```

Comment:
I was wondering why some of these parsers manually `match` instead of using the combinators. I'd imagine that using something like `choice` would let you show the syntax of the command in the help view.

Reply:
Good question, I don't quite remember! Hooking up auto-complete with the parser is a nice idea. The issue with the current help menu, (<kbd>?</kbd> key) is that there's not that much room, and so I can't present the options sometimes -- this may be why I opted for a match in some situations. I'm thinking about how to improve this though.

Reply:
yeah I can imagine some of the options strings getting pretty long, so it makes sense that you might want to cut them to keep the help menu readable. It does make discoverability harder though. :shrug: